### ISSUE TITLE
Dropbox download of large files (> 3-5 Gb) seems to require a paid account. 

### ISSUE THREAD

**Issue by chris-krohn** [R-Wright-1/kraken_metaphlan_comparison#1]
Hi all,

very excited to download and use the databases you kindly made available on Dropbox. I was going to start with kraken2_RefSeqV205_Complete_500GB and see how I go. 

After trying to download and repeatedly failing after ~ 3gb (`site was't available` error 410) using Chrome and Firefox, I searched in the Dropbox community for  a solution. It appears that a paid account is necessary for sufficient download `quota` based on [this](https://www.dropboxforum.com/t5/View-download-and-export/Failed-download-when-trying-to-download-large-file/td-p/673350) thread. 

This is not verified by Dropbox but if true it would be good to mention that a paid Dropbox account is required.

**Comment by R-Wright-1** [comment]
Hi there, as far as I'm aware, you shouldn't need a paid Dropbox account. When I was testing how the downloads etc worked, I was just doing this from the command line on our server and definitely wasn't logged into Dropbox at all. I think in this case, Dropbox seems to be experiencing some issues today - a colleague mentioned earlier that they were having syncing issues, and my personal Dropbox apparently can't connect at all right now. We were initially running into some quota issues, but this seemed to be on our end (and related to the "unlimited" Dropbox account that we have not actually being unlimited). So I did make a bit of a note about this on the main wiki page, but since we switched to having the largest database hosted on AWS, I haven't heard from anyone telling me that they have issues (and I was regularly getting emails prior to this).

If this persists when Dropbox otherwise seems to have sorted itself out, then I'll have a look into it! In the meantime, the largest database is hosted by AWS now (instructions on downloading are [here](https://github.com/R-Wright-1/kraken_metaphlan_comparison/wiki)), and if you have access to a FTP site or anything similar through your institution then I'd be happy upload whichever database you wanted to use to there (I've done similar for transferring intermediate database files to others). 

Robyn
